### ISSUE TITLE
Pushstr generates non-optimal instructions for single-character strings

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#319]
```
$ shellcraft i386.pushstr A | disasm
   0:   68 01 01 01 01          push   0x1010101
   5:   81 34 24 40 01 01 01    xor    DWORD PTR [esp],0x1010140
```

Realistically, this should look something like

```
$ asm 'push 0x41' | disasm
   0:   6a 41                   push   0x41
```

**Comment by IdolfHatler** [comment]
This is fixed in #302. Somebody should look at it. :/

**Comment by IdolfHatler** [comment]
Closing as #302 have been merged.
